autogluon/task/tabular_prediction/tabular_prediction.py

```diff
@@ -399,6 +399,13 @@ def fit(train_data,
                 Note: The file containing your `FeatureGenerator` class must be imported into current Python session in order to use a custom class.
             feature_generator_kwargs : dict, default={}
                 Keyword arguments to pass into the `FeatureGenerator` constructor.
+                Valid `AutoMLFeatureGenerator` kwargs:
```

Comment:
maybe provide full path to `AutoMLFeatureGenerator`

Reply:
Added